addons/website_crm_partner_assign/report/crm_partner_report.py

```diff
@@ -26,7 +25,7 @@ class CrmPartnerReportAssign(models.Model):
         'account.invoice.report': ['invoice_date', 'partner_id', 'price_subtotal', 'state', 'move_type'],
         'crm.lead': ['partner_assigned_id'],
         'res.partner': ['activation', 'country_id', 'date_partnership', 'date_review',
-                        'grade_id', 'parent_id', 'team_id', 'user_id'],
```

Comment:
Looks like we missed this one: 

https://github.com/odoo/odoo/blob/master/addons/website_crm_partner_assign/models/crm_lead.py#L85

?

Reply:
Hello @awa-odoo 

changes are done.
commit msg is also changed :sweat_smile: .
Can you please have a look?

Thanks :slightly_smiling_face: